dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/AliasHelper.java

```diff
@@ -78,9 +78,9 @@ public static WorkflowVersion getAndCheckWorkflowVersionResource(AuthenticatedRe
      * @param blockFormat if true don't allow specific formats
      * @return the Workflow Version
      */
-    public static WorkflowVersion addWorkflowVersionAliasesAndCheck(AuthenticatedResourceInterface authenticatedResourceInterface, WorkflowDAO workflowDAO,
+    public static WorkflowVersion addWorkflowVersionAliases(AuthenticatedResourceInterface authenticatedResourceInterface, WorkflowDAO workflowDAO,
```

Comment:
`authenticatedResourceInterface` and `workflowDAO` parameters are now unused, you can remove them.

Can't comment directly there in the PR, but JavaDoc could be updated -- the overview (no longer only works for the owner of the entry), the user description, and remove the unused params if you remove them from code.

Reply:
> `authenticatedResourceInterface` and `workflowDAO` parameters are now unused, you can remove them.
> 
> Can't comment directly there in the PR, but JavaDoc could be updated -- the overview (no longer only works for the owner of the entry), the user description, and remove the unused params if you remove them from code.

Sure, I updated the javadoc comment.

I didn't remove the unused params on purpose, because the changes ripple a ways up through the call stack, and I didn't want the diffs to cloud this PR, given that it's a hotfix.  I will write a ticket to clean up, which should probably happen on develop after the hotfix branch merges, could actually be best done as a slightly-bigger refactor, some of the params could change type...